Observer/RemoveBlocksObserver.php

```diff
@@ -55,11 +55,16 @@ public function __construct(
     public function execute(Observer $observer)
     {
         $layout = $observer->getLayout();
-        // When Bolt SSO is enabled, we replace the default sign in / register buttons with our own.
         if ($this->configHelper->isBoltSSOEnabled()) {
-            $layout->unsetElement('register-link');
-            $layout->unsetElement('authorization-link');
-            $layout->unsetElement('authorization-link-login');
+            // Remove native block on login page
+            $layout->unsetElement('customer_form_login');
```

Comment:
We need this code only on the login page but try to set and unset elements on each page.
Is this any way to check on which page we are before? Does it make sense?

Reply:
Done